en/django_forms/README.md

```diff
@@ -44,36 +44,48 @@ So once again we will create a link to the page, a URL, a view and a template.
 
 ## Link to a page with the form
 
-It's time to open `blog/templates/blog/base.html` in the code editor. In the `div` named `page-header`, we will add a link:
+Before we add the link, we need some icons to use as buttons for the link. For this tutorial, download [file-earmark-plus.svg](https://raw.githubusercontent.com/twbs/icons/main/icons/file-earmark-plus.svg) and save it in the folder `blog/templates/blog/icons/`
+
+> Note: To download the SVG image, open the context menu on the link (usually by right-clicking on it) and choose "Save link as" option. Now you can navigate to the desired directory in the window on your screen and save the file.
+
+It's time to open `blog/templates/blog/base.html` in the code editor. Now we can use this icon file inside the base template as follow. In the `div` named `container` inside `page-header` `div` tag, we will add a link:
```

Comment:
```suggestion
It's time to open `blog/templates/blog/base.html` in the code editor. Now we can use this icon file inside the base template as follows. In the `div` named `container` inside `page-header` `div` tag, we will add a link:
```

Is "In the `div` named `container` inside `page-header` `div` tag" comprehensible to someone who's just begun learning HTML?

Reply:
@das-g True. I only followed the pattern. Similar pattern of referencing html tags and class through out the tutorial in different parts.

So how can we rewrite the sentence in a simpler terms?

Reply:
> So how can we rewrite the sentence in a simpler terms?

We'd probably have to split it up into several sentences, maybe showing also the code before. E.g.

> It's time to open `blog/templates/blog/base.html` in the code editor. Now we can use this icon file inside the base template as follows. At the beginning of the `body` element (so, right after the tag `<body>`) you should have code like this:
> ```html
> <div class="page-header">
>     <div class="container">
>         <h1><a href="/">Django Girls Blog</a></h1>
>     </div>
> </div>
> ```
> Inside the inner `div` element (so, right after `<div class="container">`), we will add a link, just before the `<h1>...` part:

However, the explanation could remain vastly simpler, if there weren't nested `div` elements here. Are they really needed for this migration or would a single `div` element with two classes work, too? (See other comment in this review.)